circom-types/src/groth16/verification_key.rs

```diff
@@ -291,53 +288,53 @@ mod test {
             fs::read_to_string("../test_vectors/Groth16/bls12_381/verification_key.json").unwrap();
         let vk = serde_json::from_str::<JsonVerificationKey<Bls12_381>>(&vk_string).unwrap();
         let alpha_1 = test_utils::to_g1_bls12_381!(
-            "1006274644424409217186953213662503172434575368717179668374437354164299156533023899663544377042948523039106817870257",
-            "1056550161284843386841851874013894300183177686460359778532448293436751746312447247913808276245741832724930728476526"
+            "573513743870798705896078935465463988747193691665514373553428213826028808426481266659437596949247877550493216010640",
```

Comment:
why do we need new KAT vectors?

Reply:
One test we had did not use the .json verification key to verify, but the verification key in the zkey.
I wanted to mirror the actual workflow, therefore tried to read the verification key, but the verification key in the test folder was no longer the one needed for the verification. Therefore, I extracted it from the zkey, overriding the other test case.

I could just keep both, if prefered

Reply:
Nah was just a question, is fine as is